vision/unit_tests/test__gax.py

```diff
@@ -23,7 +23,63 @@ def _get_target_class(self):
     def _make_one(self, *args, **kwargs):
         return self._get_target_class()(*args, **kwargs)
 
-    def test_gax_not_implemented(self):
+    def test_ctor(self):
+        from google.cloud.gapic.vision.v1 import image_annotator_client
+
+        client = object()
+        api = self._make_one(client)
+        self.assertEqual(api._client, client)
+        self.assertIsInstance(api._api(),
+                              image_annotator_client.ImageAnnotatorClient)
+
+    def test__to_gapic_feature(self):
+        from google.cloud.vision._gax import _to_gapic_feature
```

Comment:
This is the function under test, please keep in line with our testing style and write a `Test__to_gapic_feature` class (inherits from `unittest.TestCase`)